### ISSUE TITLE
gpbackup installation instructions are out of date

### ISSUE THREAD

**Issue by baileygm** [greenplum-db/gpbackup#626]
Since go version 1.17 when the go get command was deprecated it appears that the instructions to download and install gpbackup are no longer valid.  I have tried using go install and am unable to install gpbackup

**Comment by baileygm** [comment]
The approach I have found to work is simply to clone the github repo and use go build

git clone --branch 1.27.0 --single-branch http://github.com/greenplum-db/gpbackup.git
cd gpbackup
go build -o /go/bin/gpbackup ./gpbackup.go
go build -o /go/bin/gprestore ./gprestore.go
go build -o /go/bin/gpbackup_helper ./gpbackup_helper.go

Is this valid ?

**Comment by khuddlefish** [comment]
Yeah I think I would recommend just cloning it. We can update the instructions to state that as well. Using `go install` now puts it in a location is more intended just for using a module rather than developing.
